code/modules/mob/living/blood.dm

```diff
@@ -29,13 +29,10 @@
 // Takes care blood loss and regeneration
 /mob/living/carbon/human/handle_blood()
 
-	if(NOBLOOD in dna.species.species_traits)
+	if((NOBLOOD in dna.species.species_traits) || (bleed_rate < 0))
```

Comment:
This will cause the handle_blood cycle to be skipped on negative bleed rate. Undo this change.